src/cli/telemetry/telemetry.ts

```diff
@@ -92,7 +101,15 @@ export function isUsingYarn(sys: d.CompilerSystem) {
   return sys.getEnvironmentVar('npm_execpath')?.includes('yarn') || false;
 }
 
-export async function getActiveTargets(config: d.Config): Promise<string[]> {
+/**
+ * Build a list of the different types of output targets used in a Stencil configuration.
+ *
+ * Duplicate entries will not be returned from the list
+ *
+ * @param config the configuration used by the Stencil project
+ * @returns a unique list of output target types found in the Stencil configuration
+ */
+export async function getActiveTargets(config: d.ValidatedConfig): Promise<string[]> {
```

Comment:
we don't have to change this necessarily, but any idea why this function is `async`?

Reply:
i'm gonna guess it was a copy paste error when it was orignally added.  No time like the present though I suppose, should be easy enough to fix